accepted/2021/utf8/requirements/README.md

```diff
@@ -0,0 +1,26 @@
+# Requirements
+
+**DRAFT**
+
+**PM** [Immo Landwerth](https://github.com/terrajobst)
+
+## [UTF-8 Strings should be pre-validated](validation.md)
+
+- allow us to do better optimizations, for example when enumerating runes
+- it's consistent with other ecosystems
+- better safety and reliability
+
+## String/UTF-8 String literals should be in read-only memory (read-only section of PE file)
```

Comment:
This is implementation detail, not a requirement.

Reply:
Presumably the requirement would be more like "Reduced memory consumption on a densely-packed machine"?